staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/width_test.go

```diff
@@ -87,7 +87,7 @@ func TestWorkEstimator(t *testing.T) {
 				"events.foo.bar": 699,
 			},
 			maxSeats:             10,
-			initialSeatsExpected: 8,
+			initialSeatsExpected: 7,
```

Comment:
I don't know why it changed. You have more experience with PnF so please double check it.

Reply:
This is fine - with the change this request is now assumed to be served from watchcache and the cost of it is proportional to total number of objects of that type (699).
Previously it was proportional to 2*limit (798).